core/src/main/java/org/bitcoinj/wallet/KeyChainGroupStructure.java

```diff
@@ -16,21 +16,115 @@
 
 package org.bitcoinj.wallet;
 
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.crypto.ChildNumber;
 import org.bitcoinj.crypto.HDPath;
 import org.bitcoinj.script.Script;
+import org.bitcoinj.utils.Network;
 
-/** Defines a structure for hierarchical deterministic wallets. */
+/**
```

Comment:
This comment could perhaps be a bit more comprehensible if it would first state everything for BIP32 wallets, and then everything for BIP44 wallets.

Alternatively, at least I would add: **In the BIP-43 case,** currently only BIP-44…

Reply:
Fixed and force-pushed.